lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@andy-armstrong @muzaffaryousaf 
Thank you both of you.
In my case, I tried to change this variable like
- lms/envs/common.py
```
PREVIEW_DOMAIN = 'preview2'
```
and, 
- (c/l)ms.env.json.
```
"PREVIEW_LMS_BASE": "preview2.hoge.com"
```

But Studio preview button is not working. (got publish pages, can't see preview pages)
If I use just a "preview" then, it works.
Please try this situation. I am using Cypress code.